### PR TITLE
Creating a ERC-721 Contract

### PR DESCRIPTION
**Pull request by AnmolSirola**
## Description

Making a ERC-721 smart contract which fulfills all the requirement of erc-721 standard.

Fixes #<62>

## Type of change

- [x] Add a new project

## Domain of Contribution

- [x] Project using only Solidity

- [x] Help of OpenZeppelin

## Checklist ✓

- [x] l have performed a self-review of my own code or work.

- [x] I have commented on my code, particularly in hard-to-understand areas.

- [x] My changes generate no new warnings

- [x] I'm SSOC'22 contributor

### CODE REVIEW

**Comment by Vikash-8090-Yadav** [comment]
Please upload a working video and paste in the last  of the readme @AnmolSirola

**Comment by AnmolSirola** [comment]
How should I make a working video of the contract , even if I screen record it and use all the functions of my code one by one it will be very confusing coz there are a lot of functions and I have to use multiple accounts from remix to run a particular function ,which account is being used for which function that will be a confusion too . How can solve all these issue and still make a screen record.

**Comment by Vikash-8090-Yadav** [comment]
> How should I make a working video of the contract , even if I screen record it and use all the functions of my code one by one it will be very confusing coz there are a lot of functions and I have to use multiple accounts from remix to run a particular function ,which account is being used for which function that will be a confusion too . How can solve all these issue and still make a screen record.

Use   only few function  in that which is important . As this is benificial to all the new user who is checking your code .

**Comment by AnmolSirola** [comment]
@Vikash-8090-Yadav Can u look check this PR now .

File: Level2/NFT-Smart-Contract/Readme.md [hunk 1]
```diff
@@ -30,3 +30,9 @@ Funtions in the smart contract:-
 - symbol :- Tells the symbol of our token ( The symbol of the token which we passed in our constructor)
 
 - tokenURI :-Returns the URI for a given token ID.
+
+A video of me running the important function (Note:- 1st Account starts with 0x5B8 ,2nd Account starts with 0xAb8)
```
Comment (Vikash-8090-Yadav) [inline Level2/NFT-Smart-Contract/Readme.md:34]: Make this line (34) as a heading or in a bold 
Thank u

**Comment by AnmolSirola** [comment]
@Vikash-8090-Yadav  Sir, Can u check it now

File: Level2/NFT-Smart-Contract/Readme.md [hunk 2]
```diff
@@ -7,9 +7,9 @@
 Funtions in the smart contract:-
```
Comment (Vikash-8090-Yadav) [inline Level2/NFT-Smart-Contract/Readme.md:7]: Do one last thing .

Make this line  7 as heading 
in line 8 make approve as  a bold and  do the same the rest of the lines like in line 9 make MintNfT  , safeTransferFrom etc as a bold

Comment (AnmolSirola) [reply Level2/NFT-Smart-Contract/Readme.md]: @Vikash-8090-Yadav Done Sir

**Comment by Vikash-8090-Yadav** [comment]
@AnmolSirola  Great work ...Looking forward for your contribution

**Comment by Vikash-8090-Yadav** [comment]
@AnmolSirola  update the branch to the latest .

**Comment by AnmolSirola** [comment]
Updated my branch to latest.

**Comment by Vikash-8090-Yadav** [comment]
Don't run all  the function, run only imp one.
like 2 or 3  any function .

**Pull request merged**